components/settings.gradle.kts

```diff
@@ -0,0 +1,3 @@
+include(":VideoPlayer")
```

Comment:
Maybe we keep only two modules? The purpose of ":VideoPlayer" is unclear.

":VideoPlayer:common" will be for the main library which we will publish in the maven.
And ":VideoPlayer:desktop" is for examples?

Reply:
Yep, indeed, fixed.

Reply:
We can also remove file `components/VideoPlayer/build.gradle.kts`